scripts/gen_app_partitions.py

```diff
@@ -12,24 +12,24 @@
 # when APP_SHARED_MEM is enabled.
 print_template = """
 		/* Auto generated code do not modify */
+		APP_SHARED_ALIGN;
 		. = ALIGN( 1 << LOG2CEIL(data_smem_{0}b_end - data_smem_{0}));
 		data_smem_{0} = .;
 		KEEP(*(SORT(data_smem_{0}*)))
-		. = ALIGN(_app_data_align);
```

Comment:
this looks like a bug: app_data section alignment does not have anything to do with shared app mem, IMHO

Reply:
I added this because the next section would need to app_data_aligned. not for this section actually. 
But ya i agree this should be moved out of this script.